### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by mdeniz63** [sumatrapdfreader/sumatrapdf#2475] [Ideas]
Why there is no dark mode button for reading long user guides or user manuals;
For example below site is good for dark mode configuration.
https://oak.dev/2020/05/06/dark-mode-for-sumatra-pdf/

**Comment by GitHubRulesOK** [comment]
That is NOT dark mode it is just making invert colours more problematic simply toggle eye shift (SHIFT+I) in current pre-release its fast and easy to reverse back to normal, for themes attempts were made but caused problems with odd effects and that has been discussed hundreds if not thousands of times but no-one can make those changes to suit constantly changing windows theming so default per window is the best and even that has problems with or without 3rd party skinning.
